src/ralph/operations/changemanagement/jira.py

```diff
@@ -0,0 +1,69 @@
+from datetime import timezone
+
+from dateutil.parser import parse as parse_datetime
+from django.conf import settings
+
+from ralph.operations.models import OperationStatus
+
+
+def get_title(event_data):
+    return event_data['issue']['fields']['summary']
+
+
+def get_description(event_data):
+    return event_data['issue']['fields']['description']
+
+
+def get_ticket_id(event_data):
+    return event_data['issue']['key']
+
+
+def get_operation_status(event_data):
+    status_conf = settings.CHANGE_MGMT_OPERATION_STATUSES
+    status_map = {
+        status_conf['OPENED']: OperationStatus.opened,
+        status_conf['IN_PROGRESS']: OperationStatus.in_progress,
+        status_conf['RESOLVED']: OperationStatus.resolved,
+        status_conf['CLOSED']: OperationStatus.closed,
+        status_conf['REOPENED']: OperationStatus.reopened,
+        status_conf['TODO']: OperationStatus.todo,
+        status_conf['BLOCKED']: OperationStatus.blocked
+    }
+
+    return status_map[event_data['issue']['fields']['status']['name']]
+
+
+def get_operation_name(event_data):
+    return event_data['issue']['fields']['issuetype']['name']
+
+
+def get_assignee_username(event_data):
+    try:
+        return event_data['issue']['fields']['assignee']['key']
+    except TypeError:
```

Comment:
This place has also caught me. However, if there is no assignee, event_data['issue']['fields']['assignee'] will be None. Applying [] operator to None produces a TypeError.